exercise-1.js

```diff
@@ -252,18 +375,31 @@
         // Square every number in the array.  Store the squares in a new array.
 
         //your code...
-
-
+        function squareAll(nums){
+            squaredArray = [];
+            for(let i = 0; i < nums.length; i++){
+                squared = Math.pow(nums[i],2);
+                squaredArray.push(squared);
+            }
+            return squaredArray;
```

Comment:
This solution isn't using any Array methods that I can tell.  How might this be refactored to use at least one Array method?